index.html

```diff
@@ -162,5 +162,5 @@
   </body>
 
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-  <script async type="text/javascript" src="./public/js/visseria.min.js?v=201811031102"></script>
+  <script async type="text/javascript" src="./public/js/visseria.min.js?v=201811031106"></script>
```

Comment:
Did you change this? o.o

It's fine to change, but actually we only need to change it when we release.
(Eventually, I'll add some way to make it dynamic...)